pkgs/os-specific/linux/xp-pen-drivers/deco-01-v2/default.nix

```
@@ -0,0 +1,79 @@
+{ lib
+, stdenv
+, fetchzip
+, libusb
```

`libusb` is an alias, which we're not allowed to use in nixpkgs. This makes `nixpkgs-review` fail:
```
error: Function called without required argument "libusb" at /home/ivv/.cache/nixpkgs-review/pr-182218-1/nixpkgs/pkgs/os-specific/linux/xp-pen-drivers/deco-01-v2/default.nix:4, did you mean "librsb", "libusb1" or "ibus"?
```
```suggestion
, libusb1
```